core/src/main/java/org/elasticsearch/index/IndexService.java

```diff
@@ -538,8 +537,9 @@ public void handle(ShardLock lock) {
         }
     }
 
-    public Settings getIndexSettings() {
-        return indexSettings;
+    @Override
+    public Settings indexSettings() {
```

Comment:
why don't we keep `getIndexSettings` intead of `indexSettings()` I like the getter better?